parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -6,12 +6,8 @@ ERR_FILE_WATCH_TIMEOUT=6 {{/* Timeout waiting for a file */}}
 ERR_HOLD_WALINUXAGENT=7 {{/* Unable to place walinuxagent apt package on hold during install */}}
 ERR_RELEASE_HOLD_WALINUXAGENT=8 {{/* Unable to release hold on walinuxagent apt package after install */}}
 ERR_APT_INSTALL_TIMEOUT=9 {{/* Timeout installing required apt packages */}}
-ERR_ETCD_DATA_DIR_NOT_FOUND=10 {{/* Etcd data dir not found */}}
```

Comment:
so these ETCD codes were never used?

Reply:
true. etcd is for api-server setup, in ccp side. This code was for aks-e that needs to cover both.